data/styles/clear/include/Icons.mapcss

```diff
@@ -1422,6 +1422,12 @@ node|z16-[amenity=nightclub],
 area|z16-[amenity=nightclub]
 {icon-image: nightclub-m.svg;font-size: 11;}
 
+node|z16-[amenity=stripclub],
+area|z16-[amenity=stripclub],
+node|z16-[amenity=brothel],
+area|z16-[amenity=brothel]
```

Comment:
I guess make them z17- or z18- even.
For reference, amenity-cinema is z17-, amenity-car_rental is z18-.